docs/4.x/requirements.md

```diff
@@ -104,7 +104,7 @@ These ports are used for Cluster operation and should be open between cluster no
 | 3022-3025               | SSH                                     | Teleport internal SSH control panel       |
 | 3080                    | HTTPS                                   | Teleport Web  UI                          |
 | 3008-3012               | HTTPS                                   | Internal Telekube services                |
-| 32009                   | HTTPS                                   | Telekube Cluster/OpsCenter Admin panel UI |
+| 32009                   | HTTPS                                   | Telekube Cluster/OpsCenter Control Panel UI |
```

Comment:
note: building with `make run-docs`, It's not showing older version / throwing a 404. We should fix this.